src/app/home-page/home-page.component.html

```diff
@@ -2,8 +2,9 @@
   <div class="card-container" *ngFor="let Weapon of weapons">
     <app-camo-card
       [weapon]="Weapon"
-      [progress]="progressData[Weapon.id]"
```

Comment:
You'd still need this since the Progress contains your current and total, so you can display like 3 / /6